Plugins/Flow.Launcher.Plugin.Shell/Main.cs

```diff
@@ -86,7 +86,7 @@ public List<Result> Query(Query query)
                             IcoPath = Image,
                             Action = c =>
                             {
-                                Execute(Process.Start, PrepareProcessStartInfo(m));
+                                Execute(Process.Start, PrepareProcessStartInfo(m, c.SpecialKeyState.CtrlPressed));
```

Comment:
you need to do something with this keystate right, passing it in is not going to start the shell as admin right?

Reply:
I guess the CtrlPressed is a bool (when ctrl is pressed, it is true), and this parameter is starting as admin?

Reply:
oohh ok, i didnt look at the method properly. cool